lib/node_modules/@stdlib/math/base/special/sincospi/lib/index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/**
+* Simultaneously compute the sine and cosine of a number times π.
+*
+* @module @stdlib/math/base/special/sincospi
+*
+* @example
+* var sincospi = require( '@stdlib/math/base/special/sincospi' );
+*
+* var v = sincospi( 0.0 );
+* // returns [ ~0.0, ~1.0 ]
```

Comment:
Return values need to be updated.

Reply:
Isn't that correct? `sin(pi*0) = 0` and `cos(pi*0) = 1`. Wouldn't that be correct? Perhaps with/without the tildes for this case?

Reply:
Ah right, sorry. Forget my comment.